### PR TITLE
collect-info.sh: Collect kube specific info on kubevirt eve

### PR DESCRIPTION
**Pull request by zedi-pramodh**
If it is kubevirt type eve, collect the kube specific info we need to help with debugging the issues.

### CODE REVIEW

**Review by eriknordmark — COMMENTED** [review]
Please update the version number which collect-info includes in the info it collects. It is somewhere at the top of the script.

**Comment by zedi-pramodh** [comment]
Ah missed that. Will update.

File: pkg/debug/scripts/collect-info.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #
 
 # Script version, don't forget to bump up once something is changed
-VERSION=13
+VERSION=14
```
Comment (OhmSpectator) [inline pkg/debug/scripts/collect-info.sh:8]: The current version in the master is 15, so it should be switched to 16.

Comment (zedi-pramodh) [reply pkg/debug/scripts/collect-info.sh:8]: Thanks for pointing it out. Will update.

**Comment by eriknordmark** [comment]
Once the conflict is resolved this can be merged,

**Comment by zedi-pramodh** [comment]
Resolved the conflict and committed using web editor

File: pkg/debug/scripts/collect-info.sh [hunk 2]
```diff
@@ -5,7 +5,8 @@
 #
 
 # Script version, don't forget to bump up once something is changed
-VERSION=15
+
```
Comment (milan-zededa) [inline pkg/debug/scripts/collect-info.sh:8]: Seems like an unnecessary empty line, otherwise LGTM.

**Review by eriknordmark — APPROVED** [review]
LGTM but conflict to resolve

**Comment by sergei-zededa** [comment]
I would add it makes sense to collect
kubectl get all
kubectl get events
kubectl logs - at least for important pods.

**Comment by zedi-pramodh** [comment]
> I would add it makes sense to collect
> kubectl get all
> kubectl get events
> kubectl logs - at least for important pods.

This is first version. We will keep adding as the testing progresses and we learn more.

**Comment by zedi-pramodh** [comment]
Submitted new PR https://github.com/lf-edge/eve/pull/3741 to address the conflicts. I am closing this one. Please approve and merge PR 3741.

**Pull request closed without merging**